toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -693,8 +697,10 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	hwInfo.m_type = Irix;
 #else
 
-	int mib[2];
 	TINT64 physMemSize;
+
+#ifdef MACOSX
```

Comment:
Might be better to refactor all of the `#ifdef` of this function into a single one with multiple `#elif`.